akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -1811,8 +1812,8 @@ final class Flow[-In, +Out, +Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends
   def detach: javadsl.Flow[In, Out, Mat] = new Flow(delegate.detach)
 
   /**
-   * Materializes to `Future[Done]` that completes on getting termination message.
-   * The Future completes with success when received complete message from upstream or cancel
+   * Materializes to `CompletionStage[Done]` that completes on getting termination message.
```

Comment:
Should be `CompletionStage<Done>` even? (If you don't mind boyscouting a bit more :) )